### PR TITLE
LRAC-12014 - [F] Create a table to list properties

### PR DESCRIPTION
**Pull request by ilzamcmed**
Sub-task [https://issues.liferay.com/browse/LRAC-12014](https://issues.liferay.com/browse/LRAC-12014)

### CODE REVIEW

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Comment by ilzamcmed** [comment]
ci:test:sf

**Review by fmarcio — APPROVED** [review]
:rocket: LGTM, thanks @ilzamcmed!

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/PropertiesTable.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+							<ClayTable.Cell
+								className="mr-2"
+								columnTextAlignment="end"
+							>
+								<ClayToggle
+
+									// TODO: When backend for commerce is copmpleted, create function to connect to commerce
+									// onToggle={() => setActiveCommerce(!active)}
+
+									toggled={true}
```
Comment (interaminense) [inline modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/PropertiesTable.tsx:91]: Just 
```suggestion
									toggled
```

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/PropertiesTable.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+									: property.dataSources.length}
+							</ClayTable.Cell>
+
+							<ClayTable.Cell
+								className="mr-2"
+								columnTextAlignment="end"
+							>
+								<ClayToggle
+
+									// TODO: When backend for commerce is copmpleted, create function to connect to commerce
```
Comment (interaminense) [inline modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/PropertiesTable.tsx:88]:
```suggestion
									// TODO: When backend for commerce is completed, create function to connect to commerce
```

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/PropertiesTable.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+						<ClayTable.Row key={index}>
+							<ClayTable.Cell className="table-cell-expand">
+								{property.name}
+							</ClayTable.Cell>
+
+							<ClayTable.Cell
+								className="mr-2"
+								columnTextAlignment="end"
+							>
+								{/* TODO: When backend for commerce is copmpleted, display the channels linked*/}
```
Comment (interaminense) [inline modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/PropertiesTable.tsx:69]:
```suggestion
								{/* TODO: When backend for commerce is completed, display the channels linked*/}
```

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/PropertiesTable.tsx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+									// onToggle={() => setActiveCommerce(!active)}
+
+									toggled={true}
+								/>
+							</ClayTable.Cell>
+
+							<ClayTable.Cell columnTextAlignment="end">
+								<ClayButton
+
+									// TODO: When backend for commerce is copmpleted, create function to open modal
```
Comment (interaminense) [inline modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/PropertiesTable.tsx:98]:
```suggestion
									// TODO: When backend for commerce is completed, create function to open modal
```

**Review by interaminense — CHANGES_REQUESTED** [review]
Hi @ilzamcmed, Thanks for great work on it. Just few adjustments.

Is there any language that needs to be created in this PR?

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/PropertiesTable.tsx (hunk 1 above)
Comment (ilzamcmed) [reply modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/PropertiesTable.tsx:91]: Fixed

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/PropertiesTable.tsx (hunk 4 above)
Comment (ilzamcmed) [reply modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/PropertiesTable.tsx:98]: Fixed

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/PropertiesTable.tsx (hunk 2 above)
Comment (ilzamcmed) [reply modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/PropertiesTable.tsx:88]: Fixed

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by interaminense** [comment]
Hi Ilza, could rebase it with master and then send directly to Brian?

**Pull request merged**